ODPi-Runtime.md

```diff
@@ -43,19 +54,19 @@ Each entry will have a designation (free text in square brackets) in order to pi
 
 The designation [TEST_ENVIRONMENT] is reserved for entries that are defining constraints on the environment in which ODPi-compliant Hadoop platforms are expected to run. The output of the reference implementation validation testsuite is expected to capture enough information to identify whether the test execution environment conforms to the specification.
 
-Hadoop Build Specifications
+Build Specifications
 ===========================
 
-To help achieve TONE, ODPi-compliant Hadoop platforms MUST conform to the following build specifications.
+To help achieve TONE, ODPi-compliant platforms MUST conform to the following build specifications.
```

Comment:
ODPI-compliant is used all over this section of the spec.  If we want to change that we should have a separate issue to change them all.